src/RecoveryServices/RecoveryServices.Backup.Autorest/Az.RecoveryServices.psd1

```diff
@@ -0,0 +1,24 @@
+@{
+  GUID = '37eaf44e-78f7-4a5a-82d1-f6350681704b'
+  RootModule = './Az.RecoveryServices.psm1'
+  ModuleVersion = '0.1.0'
+  CompatiblePSEditions = 'Core', 'Desktop'
+  Author = 'Microsoft Corporation'
+  CompanyName = 'Microsoft Corporation'
+  Copyright = 'Microsoft Corporation. All rights reserved.'
+  Description = 'Microsoft Azure PowerShell: RecoveryServices cmdlets'
+  PowerShellVersion = '5.1'
+  DotNetFrameworkVersion = '4.7.2'
+  RequiredAssemblies = './bin/Az.RecoveryServices.private.dll'
+  FormatsToProcess = './Az.RecoveryServices.format.ps1xml'
+  FunctionsToExport = 'Edit-AzrecoveryServicesBackupRetentionPolicyClientObject', 'Edit-AzRecoveryServicesBackupSchedulePolicyClientObject', 'Get-AzRecoveryServicesBackupPolicy', 'Get-AzRecoveryServicesPolicyTemplate', 'New-AzRecoveryServicesBackupPolicy', 'Remove-AzRecoveryServicesBackupPolicy', '*'
```

Comment:
The cmdlet name should be `Edit-AzRecoveryXXX`. `R` should be capitalized. I see in lots of places it is `r`. Can you please fix them all?

Reply:
will do